format/FlightSql.proto

```diff
@@ -817,6 +817,17 @@ enum SqlInfo {
    * - true: if invoking user-defined or vendor functions using the stored procedure escape syntax is supported.
    */
   SQL_STORED_FUNCTIONS_USING_CALL_SYNTAX_SUPPORTED = 576;
+
+  /*
+   * Retrieves a boolean value indicating whether transactions are supported for bulk ingestion. If not, invoking
+   * the method commit in the context of a bulk ingestion is a noop, and the isolation level is
+   * `arrow.flight.protocol.sql.SqlTransactionIsolationLevel.TRANSACTION_NONE`.
+   *
+   * Returns:
+   * - false: if bulk ingestion transactions are unsupported;
+   * - true: if bulk ingestion transactions are supported.
+   */
+   INGEST_TRANSACTIONS_SUPPORTED = 577;
```

Comment:
It seems that other enum values have `SQL_` prefix.
Do we need it here too?

Reply:
I think that in other cases the `SQL_` prefix indicates some support for transactions at the language level, describing the behavior of keywords such as `BEGIN`, `COMMIT`, etc.

In the case of a bulk ingestion, there may not actually be any SQL executed but the backend may still provide some isolation for partially completed ingests. That's the intent for this field in `SqlInfo`.

Reply:
OK.